src/activeobject.h

```diff
@@ -23,7 +23,16 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "irr_aabb3d.h"
 #include <string>
 
-#define ACTIVEOBJECT_TYPE_INVALID 0
+enum ActiveObjectType {
+	ACTIVEOBJECT_TYPE_INVALID = 0,
+	ACTIVEOBJECT_TYPE_TEST = 1,
+	ACTIVEOBJECT_TYPE_ITEM = 2,
```

Comment:
In fact they are very old, it's 0.3 related. Discuting with mod maintainers and nore it can be safely removed.